lib/hammer_cli_foreman.rb

```diff
@@ -22,6 +22,7 @@ def self.exception_handler_class
   require 'hammer_cli_foreman/logger'
 
   begin
+    require 'hammer_cli_foreman/compute_resource/register_compute_resources'
```

Comment:
I'm not sure we need to register the compute resources in this phase. It's only needed in host and  compute-profile command

Reply:
Returned to the previous state.